src/game.js

```diff
@@ -0,0 +1,117 @@
+import Tile from './tiles';
+
+class Game {
+  constructor(size) {
```

Comment:
I propose to add new property in constructor to track if user has won. For ex., `this.gameWon = false; ` and we can change this property when user wins

Reply:
For what?

Reply:
It can help us to track if the game is won already and should be finished